src/main/java/me/botsko/prism/database/sql/SQLIdMapQuery.java

```diff
@@ -222,8 +261,9 @@ public int mapAutoId(String material, String state) {
 
         String query = automap.replace("<prefix>", prefix);
 
-        if (state.equals("0") || state.equals("[]"))
+        if (state.equals("0") || state.equals("[]")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)